src/database/firestore.ts

```diff
@@ -0,0 +1,37 @@
+import { Firestore, DocumentReference } from "@google-cloud/firestore";
+
+export class FirestoreDB {
+    firestore: Firestore;
+    player_collection: DocumentReference;
+
+    constructor(projectId: string) {
+        this.firestore = new Firestore({
+            projectId: projectId,
+            keyFilename: process.env.GOOGLE_APPLICATION_CREDENTIALS,
+        });
+        this.player_collection = this.firestore.collection("players").doc("rocket_league");
+
+    }
+
+    set_player(username: string, user_id: string): void {
+        this.player_collection.set({
+            username: username,
+            user_id: user_id
+        });

```

Comment:
use update to partially update since we will potentially add more players under this?